xml/System.Text.Json/Utf8JsonWriter.xml

```diff
@@ -174,8 +183,16 @@ The `CurrentDepth` property tracks the recursive depth of the nested objects / a
       </ReturnValue>
       <Parameters />
       <Docs>
-        <summary>To be added.</summary>
-        <remarks>To be added.</remarks>
+        <summary>Commits any leftover JSON text that has not yet been flushed and releases all resources used by the current instance.</summary>
+        <remarks>
```

Comment:
Are these remarks actually correct. Typically, Dispose closes the underlying stream, so advancing is not an issue, and reuse of the stream or buffer throws an ObjectDisposedException.

Reply:
@ahsonkhan can you confirm the remarks are describing the correct behavior?

@rpetrusha the remarks were correctly ported from the triple slash comments:

https://github.com/dotnet/corefx/blob/2fc59abd09c990121526360d90810e19723cb6cb/src/System.Text.Json/src/System/Text/Json/Writer/Utf8JsonWriter.cs#L313-L324

Reply:
Note: The tool detected a &lt;see cref="... inside the remark and automatically converted it to &lt;xref:

Reply:
> Are these remarks actually correct. 

Yes, at least given the implementation and intent of the type.

Since the `Utf8JsonWriter` has two modes and is re-settable (if you call the Reset without any parameters), it does not take ownership of the underlying stream or dispose of it when Dispose is called (in stream mode). In the case of the `IBufferWriter` mode, there is no concept of dispose at all, so the remark is just stating what dispose on the writer does to the underlying output sink (whether `Stream` or `IBufferWriter`).

> Typically, Dispose closes the underlying stream, so advancing is not an issue, and reuse of the stream or buffer throws an ObjectDisposedException.

We had discussed adding a `leaveOpen` (or `dispose`) bool to the ctor in the future (like [StreamReader](https://apisof.net/catalog/System.IO.StreamReader..ctor(Stream,Encoding,Boolean,Int32,Boolean))), which would dispose the stream if set to false.